Orange/widgets/unsupervised/owlouvainclustering.py

```diff
@@ -135,6 +135,9 @@ def __init__(self):
         )  # type: QWidget
 
     def _invalidate_pca_projection(self):
+        if not self.apply_pca:
+            return
+
         self.pca_projection = None
```

Comment:
This line should be moved before the check and early return. Right? Toggling the apply_pca will only invalidate the partial results starting with the graph.

Reply:
You're right of course.